homeassistant/components/cover/wink.py

```diff
@@ -35,21 +35,24 @@ def async_added_to_hass(self):
         self.hass.data[DOMAIN]['entities']['cover'].append(self)
 
     def close_cover(self, **kwargs):
-        """Close the shade."""
+        """Close the cover."""
         self.wink.set_state(0)
 
     def open_cover(self, **kwargs):
-        """Open the shade."""
+        """Open the cover."""
         self.wink.set_state(1)
 
     def set_cover_position(self, position, **kwargs):
-        """Move the roller shutter to a specific position."""
+        """Move the cover shutter to a specific position."""
         self.wink.set_state(float(position)/100)
 
     @property
     def current_cover_position(self):
-        """Return the current position of roller shutter."""
-        return int(self.wink.state()*100)
+        """Return the current position of cover shutter."""
+        if self.wink.state() is not None:
+            return int(self.wink.state()*100)
+        else:
+            return STATE_UNKNOWN
```

Comment:
This should return `None` if not known.

Reply:
Oh okay, should I open a PR to fix that and request it get added to the 0.61 PR or just have it go in the following release?

Reply:
No rush. Make PR when you have time.